kernel/advanced_cmds/fortune.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+#include "../../drivers/display.h"
+
+void ksh_fortune() {
+  const char fortunes[3] = {"Pohl's Law: Nothing is so good that somebody, somewhere, will not hate it.", "You either die a smart fella, or live long enough to become a fart smella", "Everyone asked you about your favorite dinosaur as a kid, now, nobody cares", "Even stock traders want to arrest hatred", "Generically, fiery liquors that produce madness in total abstainers.", "Our remedies oft in ourselves do lie,? Which we ascribe to heaven.", "One of the most conspicuous qualities of a man in security."};
+  kprintc("\n");
```

Comment:
// You are passing a `"string"` on line 6, 8 and 9 as parameter, but `kprintc()` only accepts `'char'`. If you want to print strings, use  `kprints()`, defined in [kernel/io.h](https://github.com/GandelXIV/pidi-os/blob/main/kernel/io.h).